### PR TITLE
[Android] Fix FlowDirection for Labels

### LINKED ISSUE #7512: RTL CollectionView looks odd
### Description
Looks weird:
image (1)

### Steps to Reproduce

https://docs.microsoft.com/en-us/samples/xamarin/xamarin-forms-samples/userinterface-collectionviewdemos/

Nav to page as shown in description. Observe left alignment as in description.

### Expected Behavior

Expected it to be right aligned. Expected it to mirror of LTR.

### Actual Behavior

Is left aligned. 

### Basic Information

Android 9.0 on Master.

### PR DESCRIPTION
**Pull request by adrianknight89**
### Description of Change ###

I was looking at #7512 and noticed the issue is not a `CollectionView` problem. If you go to `FlowDirection` gallery, you'll see that RTL flow is broken. At the level of `VisualElementRenderer`, flow direction is being applied only to the layout direction. This does not seem to be working for `TextView`. Instead, we could use a different method for that.

### Issues Resolved ### 

- fixes #7512

### Testing Procedure ###
Run the attached test. Also, go to `FlowDirection` gallery and make sure the labels are aligned properly when you change the flow direction.

### PR Checklist ###

- [x] Targets the correct branch
- [x] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by rmarinho** [comment]
Screenshot_1569508479

**Comment by samhouts** [comment]
Yeah, we have this as an enhancement (https://github.com/xamarin/Xamarin.Forms/issues/2447). Setting it explicitly like this should bring iOS and Android in line with each other a the moment, but I had planned to add a new core property for TextDirection so it could be overridden. Not all labels should have the text direction set explicitly like this (my interpretation of the [guidelines](https://material.io/design/usability/bidirectionality.html#) is that English _should_ be left-to-right), so I was not ready to force it. However, I'm willing to accept that my interpretation was incorrect. :)

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7512.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+			BindingContext = new ViewModel7512();
+		}
+#endif
+
+		protected override void Init()
+		{
+
+		}
+
+		private void HandleButtonClick(object sender, EventArgs e)
```
Comment (samhouts) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7512.xaml.cs]:
```suggestion
		void HandleButtonClick(object sender, EventArgs e)
```

**Comment by adrianknight89** [comment]
@samhouts If the current locale is Arabic or Hebrew, then the developer can enforce a RTL flow whereas English and most other languages should follow LTR. 

> Text should always be in the correct direction for the language it’s in. 

As you said, the recommendation for English would be LTR (i.e. no mirroring). Perhaps, the right way to use `FlowDirection` is in code based on the current locale rather than XAML for all locales.

I actually haven't used `FlowDirection` in my code yet. I assumed that Android and iOS would be smart enough to handle this for us. Is that not the case?

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7512.xaml.cs (hunk 1 above)
Comment (rmarinho) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7512.xaml.cs]:
```suggestion
	 void HandleButtonClick(object sender, EventArgs e)
```

**Pull request merged**